haas/ext/switches/_console.py

```diff
@@ -74,6 +74,11 @@ def disconnect(self):
         """End the session. Must be at the main prompt."""
 
     def _set_native(self, old_native, network_id, interface):
+        """This method is intended to be over written.
+
+        In some switches, we need to create a vlan before setting that as
+        native. `interface` is needed to get back to the interface prompt
+        to set the native vlan for that interface"""
```

Comment:
Honestly, I think the problem is that the control flow is somewhat slapped together. The console code I wrote had a particular way of managing the prompts, which this doesn't fit with, and that mismatch is showing up in the function signatures.

Question: can we just put in the setup docs for the switch that you have to create all of the vlans you intend on using for HIL, and cut out this step entirely?

(Sorry for letting this fall through the cracks).